docker-init.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/sh
 echo "prepare environment"
 # replace BASE_URL constant by REMARK_URL
-sed -i "s|https://demo.remark42.com|${REMARK_URL}|g" /srv/web/*.js
+sed -i "s|https://demo.remark42.com|${REMARK_URL}|g" /srv/web/*.{js,html}
+
+if [ -n "${SITE_ID}" ]; then
+  #replace "site_id: 'remark'" by SITE_ID
+  se -i "s|'remark'|'${SITE_ID}'|g" /srv/web/*.html
```

Comment:
Also, I think `SITE_ID` could be passed via env vars on build time without replacing it after.

Reply:
Yes, if I will build my own docker image this pull request not needed. But I use public docker image`umputun/remark42` and this changes will fix it for usage without  subdomain.

Reply:
Replacing `remark` could cause side-effects like replacing this word in unexpected palaces.
I could suggest putting unique value or template like expression to places that we want to replace.

Something like `{% SITE_ID %}` and comment about how it works.
The same we should do for replacing `REMARK_URL`

Reply:
Yes, current solution is dirty hack. And `site_id` is hard coded. But this script called only inside docker container. Would not this solution break the usage without docker on bare metal?

If I will add expressions like `{% SITE_ID %}` and `{% REMARK_URL %}` it will keep the same after `npm run build`. And replace expressions to correct values will require `postbuild` script with `sed` commands.

Reply:
Yep, you are right. And now we have the issue in bin build. With this PR we will fix the issue only for docker.
Let's use your fix and I will think about how to fix it in all ways for all builds.